lib/vagrant-vbguest/installers/linux.rb

```diff
@@ -166,21 +175,21 @@ def systemd_tool
         end
       end
 
-      # Checks for the correct location of the 'vboxadd' tool.
+      # Checks for the correct location of the tool provided.
       # It checks for a given list of possible locations. This list got
       # extracted from the 'VBoxLinuxAdditions.run' script.
       #
-      # @return [String|nil] Absolute path to the +vboxadd+ tool,
+      # @return [String|nil] Absolute path to the tool,
       #                      or +nil+ if none found.
-      def vboxadd_tool
+      def find_tool(tool)
```

Comment:
Hi, do you think it's safe to assume, that if we found `vboxadd` in any path, that the same path will also include `vboxadd-service`? If possible, I'd like to cache the "too path" and avoid running that inline shell script twice.